model_analyzer/analyzer.py

```diff
@@ -112,7 +112,8 @@ def profile(self, client, gpus):
         # Profile all models concurrently
         if self._should_profile_multiple_models_concurrently():
             try:
-                self._model_manager.run_models(
+                # FIXME TKG need new function for this
```

Comment:
need to resolve this fixme. Just put an assert here for now? Will be added back in by 518